src/bidiMapper/bidiMapper.ts

```diff
@@ -21,7 +21,7 @@
  * TODO: eslint rule for this.
  */
 
-export {CdpConnection, CdpClient} from './CdpConnection.js';
 export {BidiServer} from './BidiServer.js';
 export {BidiTransport} from './BidiTransport.js';
+export {CdpClient, CdpConnection} from './CdpConnection.js';
```

Comment:
Unless it's done by automatic tool (and I believe it is not), I'd avoid this import re-arrangements, as it will end up with reformat mess.